src/runner/browser-job.js

```diff
@@ -117,10 +117,9 @@ export default class BrowserJob extends AsyncEventEmitter {
             // NOTE: before hook for test run fixture is currently
             // executing, so test run is temporary blocked
             const isBlocked             = this.testRunControllerQueue[0].blocked;
-            const isConcurrency         = this.opts.concurrency > 1;
             const hasIncompleteTestRuns = this.completionQueue.some(controller => !controller.done);
 
-            if (isBlocked || hasIncompleteTestRuns && !isConcurrency)
+            if (isBlocked || hasIncompleteTestRuns)
```

Comment:
this `!isConcrrency` condition is required now for this test: `testcafe\test\functional\fixtures\concurrency\testcafe-fixtures\concurrent-test.js`